subworkflows/local/hicpro_mapping.nf

```diff
@@ -50,7 +50,8 @@ workflow HICPRO_MAPPING {
   BOWTIE2_ALIGN(
     ch_reads,
     index.collect(),
-    Channel.value(true).collect()
+    true,
+    false
```

Comment:
To keep track of what the booleans are doing, I usually assign them to a local variable or at least add some comments, could be good for other contributors.

Reply:
I added some comments. Thanks